### ISSUE TITLE
[github::postIssue] Error: Request failed with status code 422

### ISSUE THREAD

**Issue by HealthScreeningBot-Github** [HealthScreening/HealthScreeningBot#259]
# Error Details
- Error Type: **github::postIssue**
- Error Name: **Error**
- Error Message: **Request failed with status code 422**
- Error Occured At: **Friday, December 24, 2021, 11:52:02 PM Eastern Standard Time**

## Error Stack Trace

```
Error: Request failed with status code 422
    at createError (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/node_modules/axios/lib/adapters/http.js:293:11)
    at IncomingMessage.emit (node:events:402:35)
    at endReadableNT (node:internal/streams/readable:1343:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

## Error Metadata

```json
{
    "reqData": {
        "body": "# Error Details\n- Error Type: **interaction::commandInteractionAutocomplete**\n- Error Name: **DiscordAPIError**\n- Error Message: **Invalid Form Body\ndata.choices[0].value: Choice values must be of type \"integer\"\ndata.choices[1].value: Choice values must be of type \"integer\"\ndata.choices[2].value: Choice values must be of type \"integer\"\ndata.choices[3].value: Choice values must be of type \"integer\"\ndata.choices[4].value: Choice values must be of type \"integer\"\ndata.choices[5].value: Choice values must be of type \"integer\"\ndata.choices[6].value: Choice values must be of type \"integer\"\ndata.choices[7].value: Choice values must be of type \"integer\"\ndata.choices[8].value: Choice values must be of type \"integer\"\ndata.choices[9].value: Choice values must be of type \"integer\"\ndata.choices[10].value: Choice values must be of type \"integer\"\ndata.choices[11].value: Choice values must be of type \"integer\"\ndata.choices[12].value: Choice values must be of type \"integer\"\ndata.choices[13].value: Choice values must be of type \"integer\"\ndata.choices[14].value: Choice values must be of type \"integer\"\ndata.choices[15].value: Choice values must be of type \"integer\"\ndata.choices[16].value: Choice values must be of type \"integer\"\ndata.choices[17].value: Choice values must be of type \"integer\"\ndata.choices[18].value: Choice values must be of type \"integer\"\ndata.choices[19].value: Choice values must be of type \"integer\"\ndata.choices[20].value: Choice values must be of type \"integer\"\ndata.choices[21].value: Choice values must be of type \"integer\"\ndata.choices[22].value: Choice values must be of type \"integer\"\ndata.choices[23].value: Choice values must be of type \"integer\"\ndata.choices[24].value: Choice values must be of type \"integer\"**\n- Error Occured At: **Friday, December 24, 2021, 11:51:45 PM Eastern Standard Time**\n\n## Error Stack Trace\n\n```\nDiscordAPIError: Invalid Form Body\ndata.choices[0].value: Choice values must be of type \"integer\"\ndata.choices[1].value: Choice values must be of type \"integer\"\ndata.choices[2].value: Choice values must be of type \"integer\"\ndata.choices[3].value: Choice values must be of type \"integer\"\ndata.choices[4].value: Choice values must be of type \"integer\"\ndata.choices[5].value: Choice values must be of type \"integer\"\ndata.choices[6].value: Choice values must be of type \"integer\"\ndata.choices[7].value: Choice values must be of type \"integer\"\ndata.choices[8].value: Choice values must be of type \"integer\"\ndata.choices[9].value: Choice values must be of type \"integer\"\ndata.choices[10].value: Choice values must be of type \"integer\"\ndata.choices[11].value: Choice values must be of type \"integer\"\ndata.choices[12].value: Choice values must be of type \"integer\"\ndata.choices[13].value: Choice values must be of type \"integer\"\ndata.choices[14].value: Choice values must be of type \"integer\"\ndata.choices[15].value: Choice values must be of type \"integer\"\ndata.choices[16].value: Choice values must be of type \"integer\"\ndata.choices[17].value: Choice values must be of type \"integer\"\ndata.choices[18].value: Choice values must be of type \"integer\"\ndata.choices[19].value: Choice values must be of type \"integer\"\ndata.choices[20].value: Choice values must be of type \"integer\"\ndata.choices[21].value: Choice values must be of type \"integer\"\ndata.choices[22].value: Choice values must be of type \"integer\"\ndata.choices[23].value: Choice values must be of type \"integer\"\ndata.choices[24].value: Choice values must be of type \"integer\"\n    at RequestHandler.execute (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/node_modules/discord.js/src/rest/RequestHandler.js:349:13)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async RequestHandler.push (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/node_modules/discord.js/src/rest/RequestHandler.js:50:14)\n    at async AutocompleteInteraction.respond (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/node_modules/discord.js/src/structures/AutocompleteInteraction.js:95:5)\n    at async afterTimeAutocomplete (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/src/commands/error/log/view/autocomplete/afterTime.js:41:5)\n    at async commandInteractionAutocomplete (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/src/client/interactions/commandInteractionAutocomplete.js:32:17)\n    at async HealthScreeningBotClient.oninteractionCreate (/Users/aoyansarkar/WebstormProjects/HealthScreeningBot/src/client/extraClient.js:125:28)\n```\n\n## Error Metadata\n\n```json\n{\n    \"user\": \"199605025914224641\",\n    \"guild\": \"919616827338924042\",\n    \"channel\": \"919616827338924045\",\n    \"command\": {\n        \"base\": \"error\",\n        \"subcommand\": \"view\",\n        \"subcommandGroup\": \"log\"\n    },\n    \"arguments\": [\n        {\n            \"name\": \"log\",\n            \"type\": \"SUB_COMMAND_GROUP\"\n        }\n    ]\n}\n```\n\n# Issue Details\n- Posted By: **@PythonCoderAS**\n- Posted At: **Friday, December 24, 2021, 11:52:02 PM Eastern Standard Time**",
        "title": "[interaction::commandInteractionAutocomplete] DiscordAPIError: Invalid Form Body\ndata.choices[0].value: Choice values must be of type \"integer\"\ndata.choices[1].value: Choice values must be of type \"integer\"\ndata.choices[2].value: Choice values must be of type \"integer\"\ndata.choices[3].value: Choice values must be of type \"integer\"\ndata.choices[4].value: Choice values must be of type \"integer\"\ndata.choices[5].value: Choice values must be of type \"integer\"\ndata.choices[6].value: Choice values must be of type \"integer\"\ndata.choices[7].value: Choice values must be of type \"integer\"\ndata.choices[8].value: Choice values must be of type \"integer\"\ndata.choices[9].value: Choice values must be of type \"integer\"\ndata.choices[10].value: Choice values must be of type \"integer\"\ndata.choices[11].value: Choice values must be of type \"integer\"\ndata.choices[12].value: Choice values must be of type \"integer\"\ndata.choices[13].value: Choice values must be of type \"integer\"\ndata.choices[14].value: Choice values must be of type \"integer\"\ndata.choices[15].value: Choice values must be of type \"integer\"\ndata.choices[16].value: Choice values must be of type \"integer\"\ndata.choices[17].value: Choice values must be of type \"integer\"\ndata.choices[18].value: Choice values must be of type \"integer\"\ndata.choices[19].value: Choice values must be of type \"integer\"\ndata.choices[20].value: Choice values must be of type \"integer\"\ndata.choices[21].value: Choice values must be of type \"integer\"\ndata.choices[22].value: Choice values must be of type \"integer\"\ndata.choices[23].value: Choice values must be of type \"integer\"\ndata.choices[24].value: Choice values must be of type \"integer\"",
        "labels": [
            "bug"
        ],
        "assignees": [
            "PythonCoderAS"
        ]
    }
}
```

# Issue Details
- Posted By: **@PythonCoderAS**
- Posted At: **Friday, December 24, 2021, 11:54:35 PM Eastern Standard Time**

**Comment by PythonCoderAS** [comment]
Cause: There was an error creating your Issue: title is too long (maximum is 256 characters).

**Comment by PythonCoderAS** [comment]
Fixed in #261
